### PR TITLE
Fully support Icinga 2's native web interface alongside thruk

### PR DESCRIPTION
**Pull request by jrha**
* Move server definitions to a new config section
* Provide default URL patterns for Nagios (Thruk) and Icinga2

### CODE REVIEW

**Comment by jrha** [comment]
Rebased.

File: node/node-nagios.inc.php [hunk 1]
```diff
@@ -148,7 +159,11 @@ function detail($node, $short)
                     echo '<td>'.$dt_user.'</td>';
                     echo '<td class="fixed">'.$dt_reason."</td>";
                     echo '<td>';
-                    echo '<a class="delete" href="'.sprintf($NAGIOS_URL, $server).'/cgi-bin/cmd.cgi?cmd_typ=78&amp;down_id='.$dt_id.'" />';
+                    if (strpos($server, 'icinga') !== -1) {
+                        echo '&hellip;';
```
Comment (willfurnell) [inline node/node-nagios.inc.php:163]: There should be a way to delete downtimes from Icinga - looks like this is not a simple URL but rather a form. Don't know whether it's worth looking into this more?

**Comment by willfurnell** [comment]
Checking for icinga in the hostname should work for finding Icinga servers, but I'm not sure if there's maybe a slightly better way to do it? Otherwise happy to approve.

File: node/node-nagios.inc.php (hunk 1 above)
Comment (jrha) [reply node/node-nagios.inc.php:163]: IIRC it's more an issue that the previous method worked well due to the use of X509 auth, whereas more often than not Icinga2 dumps you on a login page and the context is lost.

**Comment by jrha** [comment]
I know, it's a bit of hack.

File: node/node-nagios.inc.php (hunk 1 above)
Comment (willfurnell) [reply node/node-nagios.inc.php:163]: Ah I see, in which case I'll resolve this.

**Review by willfurnell — APPROVED** [review]
Approved - noting the slight hack to find out if a server is Icinga or not. I can't see this breaking for now.

**Pull request merged**